docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+# Setting up a pipeline for images
+
+## Objectives
+
+* Get familiarity on how to use multiple cloud services together.
+* Get familiarity on how to use cloud services using their command-line interfaces.
+
+The tutorial focuses on the following services:
+
+* Allas
+* cPouta
+* Pukki
```

Comment:
Please add a short explanation of what is each service like.
Also add  link to the main page of each service. The more linked together is the documentation, the better.
Like:

 "[Allas](/data/allas/index.md), our object storage service. We will use it for storing the files".

Reply:
Fixed in https://github.com/CSCfi/csc-user-guide/pull/2424/commits/6dd0e754b0bd7109448f9375cfa5971b78ef747c